src/cljs/org/broadinstitute/firecloud_ui/page/workspaces_list.cljs

```diff
@@ -208,30 +208,41 @@
                        assoc :error-message status-text)))}))})
 
 
-(defn- get-workspace-id-from-nav-segment [segment]
-  (when-not (clojure.string/blank? segment)
-    (let [[ns n] (clojure.string/split segment #":")]
-      {:namespace ns :name n})))
+(defn- create-breadcrumbs-from-hash [hash]
+  (let [segments (split hash #"/")]
+    (map-indexed
+      (fn [index segment]
+        (case index
+          0 {:text "Workspaces" :href "#workspaces"}
+          1 {:text (replace (js/decodeURIComponent segment) ":" "/")}
```

Comment:
Before I special-cased out that segment and got rid of the link, you could click on that, and it would nav you to workspaces/namespace:name, without the "/Summary" on the end.  It would still work--you could click on tabs and the problem would correct itself, and if you bookmarked that URL and hit it it would fix on load (redirecting to /Summary).

You can always click on the tab, so I thought having that crumb navigable was just causing more problems than it was worth.